app/styles/custom-utilities/_text.scss

```diff
@@ -17,3 +17,35 @@
 .vlc-u-font-weight-normal {
   font-weight: normal;
 }
+
+.vlc-u-font-weight-500 {
+  font-weight: 500;
+}
+
+.vlc-u-font-color-dove-gray {
+  color: $dove-gray;
+}
+
+.vl-vi-check-thin::before {
+  padding-right: 0.5rem;
+}
+
+.vl-vi-question-mark::before {
+  padding-right: 0.5rem;
+}
+
+.vl-vi-cross::before {
+  padding-right: 0.5rem;
+}
+
+.vl-vi::before {
+  font-weight: 700;
+}
```

Comment:
Wait, you just added these styles globally across the entire app for just these icons? These aren't actually util classes per definition. Also all the utils are missing the `!important` that should be added.

To make life easier I would try to refactor away from utils. I wanna do this paired if you want to.